Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -671,9 +682,18 @@ script:
     - contextPath: Gmail.Headers
       description: All headers of specific mail (list).
       type: unknown
-    - contextPath: Gmail.Mailbox
-      description: The Gmail Mailbox.
-      type: string
+    - contextPath: Gmail.Mailboxes
+      description: The Gmail mailboxes in which emails matching the query were found.
+      type: String
+    - contextPath: Gmail.UnsearchedAcounts
+      description: The unsearched acounts with the error.
```

Comment:
Please consult with @ShirleyDenkberg on this term

Reply:
@israelpoli - can you please explain what you mean in the description "The unsearched accounts with the error." - what error? Is this supposed to be a list of Gmail accounts? Also, the contextPath should be spelled Gmail.UnsearchedAccounts (accounts with 2 c's)